### PR TITLE
Bump hc ab02f36c87999d42026b7429164ded503bb39853

### PR DESCRIPTION
**Pull request by timotree3**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -26,9 +26,8 @@ test-all:
 
 update-hc:
 	make HC_REV=$(HC_REV) update-hc-sha
-	git add nixpkgs.nix
-	git commit -m hc-rev:$(HC_REV)
-	git push origin HEAD
```
Comment (zo-el) [inline Makefile:31]: Is there a reason this is removed?

Comment (timotree3) [reply Makefile:31]: well `nixpkgs.nix` doesn't exist. I also didn't think it made much sense for a make script to make git commits on your behalf. For example, I could run this command on develop planning to check out a different branch with the changes and then all the sudden it pushes a commit to develop on my behalf

Comment (zo-el) [reply Makefile:31]: The intention of this script is meant for a bigger reason. It does not just update the values but eventually, make a release too, So the script at least has to create a new branch and push up.

Comment (zo-el) [reply Makefile:31]: The nixpkgs,nix was the old nix file, it is updated to default.nix now. It must have been an oversite.

Comment (timotree3) [reply Makefile:31]: Another update script I looked at didn't make git changes. I think it makes more sense to have the git changes as part of a separate make command. How do you want to proceed? Can we just merge this as is and add back the git stuff when we know how we're using it?

Comment (zo-el) [reply Makefile:31]: Sure, let's do that

**Review by zo-el — APPROVED** [review]
Looks good

**Pull request merged**